topics/github/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Chris Wanstrath, PJ Hyett, Tom Preston-Werner, and Scott Chacon
+display_name: GitHub
+github_url: https://github.com/github
+logo: github.png
+released: April 10, 2008
+short_description: You’re lookin’ at it.
```

Comment:
I _may_ have stolen this lil' easter egg from the repo's actual description 😆
<img width="383" alt="github_github__You’re_lookin’_at_it_" src="https://user-images.githubusercontent.com/121322/57053185-43579c80-6c41-11e9-8e7d-fff40cd20724.png">